example/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "example",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run compile && node index.js",
+    "compile": "faucet --no-fingerprint --compact"
+  },
+  "author": "Falk Hoppe <falkhoppe81@gmail.com>",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "complate-express": "file:..",
+    "express": "^4.15.3"
+  },
+  "devDependencies": {
+    "complate-jsx": "^0.7.1"
```

Comment:
complate-jsx is essentially a meta-package which abstracts the tools required for bundling and transpiling JSX modules (i.e. faucet-pipeline-js and babel-plugin-transform-react-jsx) - since complate templates are precompiled, those tools are not required at runtime